NoteFind/Alerts/ErrorAlerts.m

```diff
@@ -21,4 +22,21 @@ + (void)showAlertWithTitle:(NSString *)title withMessage:(NSString *)message wit
     [vc presentViewController:alert animated:YES completion:nil];
 }
 
++ (void)confirmingDownload:(UIViewController *)vc withNote:(Note *)note {
```

Comment:
This seems a bit strange in this file since its not a ErrorAlert

Reply:
this is supposed to just be my alerts file, so I can change the name of the file